conjurapi/client.go

```diff
@@ -124,6 +143,11 @@ func NewClientFromEnvironment(config Config) (*Client, error) {
 		return NewClientFromKey(config, *loginPair)
 	}
 
+	oidcAuthenticater, err := OIDCAuthenticatorFromEnv()
+	if err == nil && oidcAuthenticater.AuthenticateURL != "" && oidcAuthenticater.IDToken != "" {
+		return NewClientFromOIDCAuthenticator(config, *oidcAuthenticater)
```

Comment:
This seems a bit inverse from a cursory look. I think that this `NewClientFromOIDCAuthenticator ` would be used to retrive the `authnToken` from Conjur that could then be used in `NewClientFromTokenFile` constructor or am I mistaken maybe?